crates/ruff_linter/src/fix/edits.rs

```diff
@@ -122,6 +122,28 @@ pub(crate) fn remove_unused_imports<'a>(
     }
 }
 
+/// Edits to make the specified imports explicit, e.g. change `import x` to `import x as x`.
+pub(crate) fn make_redundant_alias<'a>(
+    member_names: impl Iterator<Item = &'a str>,
```

Comment:
Yep, again, I was following the example set by `fix::edit::remove_unused_imports`, which is the only other `Edit`-producing function used by this rule. Happy to try out your suggestion, though. Thanks.

Reply:
I tried this a couple different ways and I don't think it ended up simplifying things. I don't know the rust patterns here, but it seems like taking a slice ends up requiring a lot more collections because you can't map down to subfields.